build.sbt

```diff
@@ -123,7 +123,7 @@ lazy val commonSettings = ReleasePlugin.extraReleaseCommands ++ Seq(
     "-Xfuture",
     "-Ywarn-unused-import"
   ),
-  fork in Test := true,
+  fork in Test := false,
```

Comment:
I'll revert it. I use sbt console a lot, including with debugging. Therefore I need the tests not to be forked.